src/radical/saga/adaptors/pbspro/pbsprojob.py

```diff
@@ -852,13 +847,6 @@ def _job_get_info(self, job_id, reconnect):
                 and not job_info['end_time']:
                 job_info['end_time'] = time.time()
 
-
-        # PBSPRO state does not indicate error or success -- we derive that from
-        # the exit code
-        if job_info['returncode'] not in [None, 0]:
-            job_info['state'] = api.FAILED
-
-
```

Comment:
Are we sure we don't need this anymore?

Reply:
yeah, we have the following place, which should be used to handle the state

https://github.com/radical-cybertools/radical.saga/blob/4088cb1d27841a1600a7d7854c0de23f2680eef6/src/radical/saga/adaptors/pbspro/pbsprojob.py#L839

and this would just overwrite the determined status